onnxruntime/contrib_ops/cuda/quantization/matmul_nbits.cu

```diff
@@ -288,6 +288,7 @@ bool TryMatMul4Bits(
   if (n % kColsPerThreadBlock != 0 || k % 8 != 0 || m > 1) {
     return false;
   }
+  const int kWarpSize = GPU_WARP_SIZE_HOST;
```

Comment:
Was there a reason to create a kWarpSize here that shadows the earlier declaration instead of using GPU_WARP_SIZE_HOST directly in the threads constructor?